app/src/main/java/com/android/developers/androidify/MainActivity.kt

```diff
@@ -46,8 +52,25 @@ class MainActivity : ComponentActivity() {
                         Color.Transparent.toArgb(),
                     ),
                 )
-                MainNavigation()
+                MainNavigation(isWindowNotOccluded)
```

Comment:
I'd love for us to consider how this could potentially be accessed / used without having to pass the value all the way from the Top level composable down to where its needed. Is it possible to create a modifier that does this logic? 
It would be ideal if either `Modifier.onVisibilityChanged` included this logic, or if we could expose a `Modifier.onOccluded()`, as this seems to find its way through all the composables that may not need it.

cc @lelandrichardson

Reply:
Right now I went with the path of Composition Local to avoid passing the value all the way down